sim/hunter/wotlk_items.go

```diff
@@ -9,26 +9,26 @@ import (
 	"github.com/wowsims/cata/sim/core/stats"
 )
 
-var ItemSetGladiatorsPursuit = core.NewItemSet(core.ItemSet{
-	Name: "Gladiator's Pursuit",
-	Bonuses: map[int32]core.ApplyEffect{
-		2: func(agent core.Agent) {
-			hunter := agent.(HunterAgent).GetHunter()
-			hunter.AddStats(stats.Stats{
-				stats.AttackPower:       50,
-				stats.RangedAttackPower: 50,
-				stats.Resilience:        50,
-			})
-		},
-		4: func(agent core.Agent) {
-			hunter := agent.(HunterAgent).GetHunter()
-			hunter.AddStats(stats.Stats{
-				stats.AttackPower:       150,
-				stats.RangedAttackPower: 150,
-			})
-		},
-	},
-})
+// var ItemSetGladiatorsPursuit = core.NewItemSet(core.ItemSet{
+// 	Name: "Gladiator's Pursuit",
+// 	Bonuses: map[int32]core.ApplyEffect{
+// 		2: func(agent core.Agent) {
+// 			hunter := agent.(HunterAgent).GetHunter()
+// 			hunter.AddStats(stats.Stats{
+// 				stats.AttackPower:       50,
+// 				stats.RangedAttackPower: 50,
+// 				stats.Resilience:        50,
+// 			})
+// 		},
+// 		4: func(agent core.Agent) {
+// 			hunter := agent.(HunterAgent).GetHunter()
+// 			hunter.AddStats(stats.Stats{
+// 				stats.AttackPower:       150,
+// 				stats.RangedAttackPower: 150,
+// 			})
+// 		},
+// 	},
+// })
```

Comment:
Why did this get commented?

Reply:
For some reason wotlk_items.go is still active (for the T10 bonus perhaps?) and this name clashed with the new pvp-set in cata.
I could just remove it otherwise :)

Reply:
Yeah can be removed at this point